css/index.css

```diff
@@ -1 +1,498 @@
+  font-size: 1.8rem;
+  text-decoration: none;
+  color: black;
+}
+
+.header-left a:hover {
+  background-color: black;
+  color: white;
+}
+
+.header-right {
+  padding-right: 2%;
+}
+
+.header-right img {
+  width: 100%;
+  max-width: 379px;
+}
+/*Main Area*/
+main {
+  border-top: 1px solid black;
+  border-bottom: 1px solid black;
+  padding-top: 40.5px;
+  padding-bottom: 10px;
+  justify-content: space-between;
+  letter-spacing: 0.1rem;
+  line-height: 160%;
+  width: 100%;
+}
+
+main section {
+  display: flex;
+  margin-bottom: 43px;
+}
+
+main section div {
+  /* padding-left: 1%; */
+  padding: 0 1%;
+  width: 50%;
+}
+
+main section img {
+  width: 100%;
+  max-width: 800px;
+}
+
+main h2 {
+  font-size: 1.6rem;
+}
+
+main p {
+  font-size: 1.4rem;
+}
+
+/*Footer Area*/
+footer {
+  padding-top: 40px;
+  padding-left: 1rem;
+}
+
+footer h2 {
+  font-size: 1.6rem;
+}
+
+footer span {
+  font-size: 1.6rem;
+}
+
+address {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 23px;
+}
+
+.footer-phone-email {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 52px;
+}
+
+.footer-copyright {
+  display: flex;
+  justify-content: center;
+  margin-top: 55px;
+  font-size: 1.4rem;
+}
+
+@media (max-width: 800px) {
+  .container {
+    margin: 30px auto;
+    padding: 0 2%;
+  }
+
+  /*Nav Area*/
+  .nav {
+    flex-direction: column-reverse;
+  }
+
+  .nav img {
+    margin-right: 0;
+  }
+
+  .nav nav {
+    align-self: flex-start;
+    justify-content: space-between;
+    width: 98%;
+    padding: 0 2%;
+  }
+
+  .nav nav a {
+    font-size: 1.6rem;
+  }
+
+  .nav img {
+    margin-bottom: 40px;
+  }
+
+  /*Header Area*/
+  header {
+    margin-top: 15px;
+    margin-bottom: 72px;
+  }
+
+  .header-left {
+    margin: 0 auto;
+    padding: 0;
+  }
+
+  .header-left h1 {
+    font-size: 7rem;
+    letter-spacing: 0.2rem;
+    padding-left: 0;
+    margin-bottom: 30px;
+  }
+
+  .header-left a {
+    width: 60%;
+    height: 35px;
+    margin-left: 0;
+  }
+  .header-right {
+    display: none;
+  }
+
+  /*Main Area*/
+  main {
+    border-top: 2px solid black;
+    border-bottom: 2px solid black;
+    padding-top: 35px;
+    padding-bottom: 5px;
+  }
+
+  main section {
+    margin-bottom: 32px;
+  }
+
+  .main-top p {
+    line-height: 1.8rem;
+  }
+
+  main section img {
+    height: 50%;
+  }
+
+  .main-middle_img {
+    margin-bottom: 30px;
+  }
+
+  .main-bottom {
+    justify-content: space-between;
+    padding-right: 1.5%;
+  }
+
+  .main-bottom div {
+    line-height: 1.8rem;
+    width: 23rem;
+  }
+
+  /*Footer Area*/
+  footer {
+    padding-top: 35px;
+  }
+
+  address {
+    margin-bottom: 15px;
+  }
+
+  .footer-phone-email {
+    height: 48px;
+  }
+
+  .footer-copyright {
+    margin-top: 40px;
+    font-size: 1.6rem;
+  }
+}
+
+@media (max-width: 500px) {
+  .container {
+    margin: 20px auto;
+  }
+
+  /*Nav Area*/
+  .nav {
+    padding-right: 6%;
+    width: 100vw;
```

Comment:
I think the only thing giving you a horizontal scroll bar is these vw properties. The difference between vw and percentage width is percentage width respects the container. You have a little bit of overflow at the mobile media break but other than that, this thing is flawless.